source/simplex/fe_lib.cc

```diff
@@ -69,60 +69,52 @@ namespace Simplex
         {
           // We don't really have dim = 1 support for simplex elements yet, but
           // its convenient for populating the face array
+          Assert(degree <= 2, ExcNotImplemented());
```

Comment:
What about using `AssertIndexRange`? I would like the degree to be printed in the error message.

Reply:
I would rather leave it as it is: `degree` isn't an index, and the error message shouldn't suggest that it is. The error you get with `AssertIndexRange` reads like this:
```
    << "Index " << arg1 << " is not in the half-open range [" << arg2 << ","
    << arg3 << ")."
```
I don't think that would make things easier to understand.